client/src/components/DisplayTable/DisplayTable.yaml

```diff
@@ -19,6 +19,25 @@ items_per_page_description:
 show_all:
   en: Show all
   fr: Tout afficher
+select_all:
+  en: (Select All)
+  fr: TODO
+filters_applied:
+  transform: [markdown]
+  en: _Filters applied_
+  fr: TODO
+search_column:
+  en: Search column
+  fr: TODO
+search_filter_options:
+  en: Search these filter options
+  fr: TODO
+filter_data:
+  en: Filter data
+  fr: Filtrer les données
+no_data_table:
+  en: No data available for this table. Please check any filters applied.
+  fr: Aucune donnée n'est disponible pour ce tableau. TODO
```

Comment:
Just a reminder to double check all these, make sure the wording's good, and get translation.

`select_all` is not used anymore pretty sure, can delete.